merge_and_rename_NGI_fastq_files.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env/python
+# Merges all fastq_files from each samples into one file. 
+# ARGV 1 Base directory with files to be merged
+# ARGV 2 Destination dir of the merged output files
+
+import re
+import os
+import sys
+import shutil
+def merge_files(dest_dir, fastq_files):
+    #Match NGI sample number from flowcell
+    sample_pattern=re.compile("^(.+)_S[0-9]+_.+_R([1-2])_")
+    #Remove files that already have the right name (i.e have been merged already)
+    matches=[]
+    for fastq in fastq_files:
```

Comment:
this is a really picky comment and it is more a personal opinion rather than a rule. I think that a different naming would help in making more clear what the code is doing:
```
for fastq_file in fastq_files:
     try:
           ....
           fastq_files_matched.append(fastq_file)
           ....
```